content/hardware/04.pro/shields/portenta-vision-shield/tutorials/camera-to-bitmap-sd-card/content.md

```diff
@@ -0,0 +1,371 @@
+---
+title: "Saving Images as Bitmaps Into the SD Card"
+difficulty: easy
+tags: [Camera, Bitmap, SD Card]
+description: This tutorial shows you how to capture a frame from the Portenta Vision Shield Camera module and save the output as a bitmap image.
+author: Pablo Marquínez
+hardware:
+  - hardware/04.pro/boards/portenta-h7
+  - hardware/04.pro/shields/portenta-vision-shield
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+  - cli
+---
+
+## Overview
+This tutorial shows you how to capture a frame from the Portenta Vision Shield Camera module and save the output as a bitmap image. It will allow you to see the output directly on your computer without using any third party tool.
+
+## Goals
+
+- Capturing a frame from the camera.
+- Make the bitmap binary file with the correct settings.
+- Save the bitmap on SD Card.
+- Visualize the image on your computer.
+
+### Required Hardware and Software
+
+- [Portenta H7](https://store.arduino.cc/portenta-h7)
+- Portenta Vision Shield ([LoRa](https://store.arduino.cc/portenta-vision-shield-lora) or [Ethernet](https://store.arduino.cc/portenta-vision-shield))
+- 1x USB-C cable (either USB-A to USB-C or USB-C to USB-C)
+- Micro SD card
+- Arduino IDE or Arduino-cli
+
+## Instructions
+
+### 1. The Setup
+Connect the Vision Shield to your Portenta H7 as shown in the figure. The top and bottom high density connectors are connected to the corresponding ones on the underside of the H7 board. Plug in the H7 to your computer using the USB C cable.
+
+![Connecting the Vision Shield to Portenta](assets/vs_ard_gs_attach_boards.svg)
+
+#### The Camera
+
+You will be using the **Himax HM-01B0 camera module** which has a resolution of 320 by 240 and the output data its in grayscale with 8 bits per pixel (bpp), this is important to have in mind as the `.bmp` (bitmap) format has some needed configuration depending on the data being used.
+
+Inside the sketch you can use this librarys to access the camera APIs
+```cpp
+#include "camera.h"
+#include "himax.h"
+```
+
+#### Bitmap File Format
+
+The bitmap binary file needs to contain some information in order to tell the computer some required information, for example the resolution of the picture and the bit-depth (bpp)
```

Comment:
```suggestion
The bitmap binary file needs to contain some information in order to tell the computer some required information, for example the resolution of the picture and the bit-depth (bpp). Bit depth refers to the color information stored in the image. The higher the bit depth of an image, the more colors it can store. As the bit depth increases, the file size of the image also increases because more color information has to be stored for each pixel in the image.
```